Content.Server/Radiation/RadiationPulseComponent.cs

```diff
@@ -49,6 +49,16 @@ public override bool Decay
         public float MaxPulseLifespan { get; set; } = 2.5f;
 
         [DataField("dps")]
+        public override float RadsPerSecondBase
+        {
+            get => _radsPerSecond;
+            set
+            {
+                _radsPerSecond = value;
+                Dirty();
+            }
+        }
```

Comment:
Both `RadsPerSecondBase` and `RadsPerSecond` are backed by the same value (`_radsPerSecond`). Is there supposed to be a `_radsPerSecondBase`?